dnf-docker-test/features/steps/test_behave.py

```diff
@@ -253,3 +248,17 @@ def then_package_state(context, pkgs, state):
 def then_package_state(context, exit_code):
     exit_code = int(exit_code)
     assert context.cmd_rc == exit_code
+
+@then('output line should "{state}" with "{line_start}"')
+def then_package_state(context, state, line_start):
+    counter = 0
+    for line in context.cmd_output.split('\n'):
+        if line.startswith(line_start):
+            counter += 1
+    if state == 'start':
+        assert counter > 0
+    elif state == 'not start':
+        assert counter == 0
+    else:
+        raise AssertionError('The state {} is not allowed option for Then statement (allowed start, not start'
+                             .format(state))
```

Comment:
add empty line at the end of file, please. That's a UNIX convention. You should set automatically adding empty line at the end of file in your text editor.